interface.go

```diff
@@ -27,6 +27,23 @@ type Token struct {
 	SentTime     time.Time
 }
 
+// A ClientToken is a token received by the client.
+// It can be used to skip address validation on future connection attempts.
+type ClientToken struct {
+	data []byte
+}
+
+type TokenCache interface {
+	// Get searches for a ClientToken associated with the given key.
+	// Since tokens should not be reused, it should remove the token from the cache.
```

Comment:
should not or must not?

Reply:
https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#validate-future

> A client SHOULD NOT reuse a token in different connections. Reusing a token allows connections to be linked by entities on the network path (see Section 9.5). A client MUST NOT reuse a token if it believes that its point of network attachment has changed since the token was last used; that is, if there is a change in its local IP address or network interface

Reply:
I'll change the text so that it doesn't sound like normative language: "Since tokens are not supposed to be reused, it must remove..."